src/content/en/2020/markup.md

```diff
@@ -491,47 +553,50 @@ The most popular ones stand out because they are almost twice as popular than ea
+| `width=device-width` | 231,668 | 3.65% |
 
 <figcaption>{{ figure_link(caption="<code>viewport</code> specifications, and lack thereof.", sheets_gid="1414206386", sql_file="summary_pages_by_device_and_viewport.sql") }}</figcaption>
 </figure>
 
-The results show that almost half of the pages we analyzed are using the typical viewport `content` value. Still, around 688K mobile pages (and 904K desktop pages) are entirely missing a proper `content` value for the viewport meta element, with the rest of them using an improper combination of `maximum-scale`, `minimum-scale`, `user-scalable=no`, or `user-scalable=0`.
+The results show that almost half of the pages we analyzed are using the typical viewport `content` value. Still, around 10% of mobile pages are entirely missing a proper `content` value for the viewport meta element, with the rest of them using an improper combination of `maximum-scale`, `minimum-scale`, `user-scalable=no`, or `user-scalable=0`.
 
-<p class="note">For a while now the Edge mobile browser allows to <a href="https://blogs.windows.com/windows-insider/2017/01/12/announcing-windows-10-insider-preview-build-15007-pc-mobile/">zoom into a web page to at least 500%</a>, regardless of the zoom settings defined by a web page employing the viewport meta element.</p>
+<p class="note">
+  For a while now, the Edge mobile browser allows users to zoom into a web page to <a href="https://blogs.windows.com/windows-insider/2017/01/12/announcing-windows-10-insider-preview-build-15007-pc-mobile/">at least 500%</a>, regardless of the zoom settings defined by a web page employing the viewport meta element.
+</p>
 
 ### Favicons
 
-The situation around favicons is fascinating: Favicons work with or without markup—some browsers would fall back to [looking at the domain root](https://realfavicongenerator.net/faq#why_icons_in_root)—, accept several image formats, and then also promote several dozen sizes (some tools are reported to generate 45 of them; [realfavicongenerator.net](https://realfavicongenerator.net/) would return _37_ if requested to handle every case). There is, as a side note, an issue open for the HTML spec to help [improve the situation](https://github.com/whatwg/html/issues/4758).
+The state of favicons is fascinating. Favicons work with or without markup, for example some browsers would fall back to [looking at the domain root](https://realfavicongenerator.net/faq#why_icons_in_root), browsers accept favicons in several image formats, and they are also supported in several dozen sizes. In fact, some tools are reported to generate 45 favicon sizes! For example, one favicon tool, [realfavicongenerator.net](https://realfavicongenerator.net/), would return _37_ favicon sizes if requested to handle every case. As of this time of writing, there is an [open issue](https://github.com/whatwg/html/issues/4758) for the HTML spec to help improve the situation.
```

Comment:
I get some of the editing here (break large sentences down etc.), however I find it reads, “choppy.”

For the sake of simplicity I suggest to re-establish the old version, but keep the edits of the first and last sentence. Meaning and flow changed much in the middle section.